### PR TITLE
Create maps future change

### LINKED ISSUE #41: Uniform naming convention of maps from cpm/rcm/gcm data
We need the same naming rules for the maps used in `past performance`  `future change`  `Analyse` tabs in cpm page. This also influences the options in the dropdown menu in each `.vue` file for the related tabs, and therefore corresponds to issue #37 .

It is recommended to address this issue together with issue #37.

### LINKED ISSUE #42: Rename analysis to "future change"

### LINKED ISSUE #44: Check colorbar ranges
-9 to 9 for both tas and pr seems quite large. In D3.3 report, the range is -3 to 3 for precip. Consider using a smaller range.

### LINKED ISSUE #46: Colorbar takes up a lot of space
and is the same for every image. Is there a way to make it take up less space?

### LINKED ISSUE #47: Precip unit
should be mm/day I think

### PR DESCRIPTION
**Pull request by geek-yang**
- Implement new naming convention for maps
- Create maps for future change
- Use proper colorbard
- Use compact figure size
This PR closes #41, closes #42, fixes #44, closes #46 and fixes #47

### CODE REVIEW

**Comment by Peter9192** [comment]
@geek-yang is this ready? If so, can you mark it ready for review, and request a review?

**Comment by geek-yang** [comment]
This PR is ready for review. Since there are too many maps, to make it easier for review, I created another tab called "future change" and this allows you to check those maps. Note that this tab will be nested in the "past performance" one as planned. We will handle it when addressing issue #43.

The sonarcloud complain is due to the duplication of tabs. We can ignore it for now.

**Comment by geek-yang** [comment]
The notebook used to generate all these maps is [this](https://github.com/eucp-project/data-catalogue/blob/maps-future-changes/python/performance_plots.ipynb). Since there are many exceptional cases (that's why there are many `if` statement in the code) in `past performance` and `future change` and they are therefore separated in this notebook.

**Comment by bvreede** [comment]
I don't know if this works, but [this](https://github.com/eucp-project/data-catalogue/pull/50/files?file-filters%5B%5D=.ipynb&file-filters%5B%5D=.vue&show-deleted-files=true&show-viewed-files=true#diff-d08d1ef7858f392ec85792e949df4a703135e2794810f959964396dfffd5af9f) is the URL that directly refers to the PR review sans .png files.

**Review by bvreede — APPROVED** [review]
Looks good to me :)

**Comment by Peter9192** [comment]
Reviewed live yesterday, will wait for new changes.

File: components/cpm/Performance.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <div class="w-full h-full flex flex-wrap">
       <div class="w-1/3 h-full">
         <p class="pt-6 text-center text-lg prose">
-          CPM experiment
+          High-resolution models
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply components/cpm/Performance.vue:21]: Corrected!

File: components/cpm/Performance.vue [hunk 2]
```diff
@@ -27,7 +27,7 @@
       </div>
       <div class="w-1/3 h-full">
         <p class="pt-6 text-center text-lg prose">
-          RCM experiment
+          Regional models
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply components/cpm/Performance.vue:30]: Corrected!

File: components/cpm/Performance.vue [hunk 3]
```diff
@@ -36,7 +36,7 @@
       </div>
       <div class="w-1/3 h-full">
         <p class="pt-6 text-center text-lg prose">
-          GCM experiment
+          Global models
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply components/cpm/Performance.vue:39]: Corrected!

**Comment by geek-yang** [comment]
@bvreede Thanks for your review, changes that you suggested have been implemented now.

@Peter9192 Thanks for spotting the mistake about the calculation. Now all maps are calculated as "future runs (rcp85)" minus "past runs (his)". All maps are regridded to 0.5 deg (otherwise the clipping tool does not work). This PR is ready for review again.

**Comment by Peter9192** [comment]
Hey @geek-yang thanks again for the hard work. I had a quick look and it seems better now. There might still be some tweaking and maybe some more bugs, but at this stage I think it's good to merge and share it with more people - then we're more likely to catch any remaining problems.

One small thing I noticed is that some maps seem to have coordinates between 0-360 instead of -180-180. For Europe, this leads to images like this:

image

Perhaps this can be addressed in a future PR.

**Review by Peter9192 — APPROVED** [review]

**Comment by geek-yang** [comment]
> Hey @geek-yang thanks again for the hard work. I had a quick look and it seems better now. There might still be some tweaking and maybe some more bugs, but at this stage I think it's good to merge and share it with more people - then we're more likely to catch any remaining problems.
> 
> One small thing I noticed is that some maps seem to have coordinates between 0-360 instead of -180-180. For Europe, this leads to images like this:
> 
> image
> 
> Perhaps this can be addressed in a future PR.

Thanks for your review @Peter9192 . I create a new issue for it #55 . We can address it later. I will merge this PR.

**Pull request merged**